.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+    'plugin:@typescript-eslint/recommended',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: ['react', '@typescript-eslint'],
+  rules: {
+    indent: ['error', 2],
+    semi: ['error', 'always'],
+    'import/prefer-default-export': 'off',
+    'react/jsx-one-expression-per-line': 'off',
+    'react/jsx-filename-extension': 'off',
```

Comment:
```suggestion
    'react/jsx-filename-extension': [1, { 'extensions': ['.jsx', '.tsx'] }]
```

jsx를 사용하는 파일의 확장자를 jsx로 지키는 룰인데, 끄기보다는, tsx도 허용해주는 쪽이 좋을 듯 합니다
(tsx도 되는지는 모름)